feature/experimental/policy/policy_vrf_selection/otg_tests/DSCP_Based_traffc_steering_between_VRFs_and_GUE_Encap/README.md

```diff
@@ -0,0 +1,324 @@
+    * Run post test Health checks above to ensure no anomalies<br><br><br>
+
+    
+**RT-3.32.2: BE1 traffic migration to Default VRF, GUE encap and then route to ATE2:Port3**
+  * Please run all the health checks suggested above prior to starting the test as a baseline.
+  * DUT has the following VRF selection policy
+    * Statement1: traffic matching tunnel-destination "IPv4Prefix12/28", Punt to default vrf
+    * Statement2: traffic matching DSCP BE1, Punt to default vrf
+    * Statement3: traffic matching DSCP AF1, Punt to Non-default vrf
+    * Statement4: traffic matching DSCP AF2, Punt to Non-default vrf
+    * Statement5: traffic matching DSCP AF3, Punt to Non-default vrf
+    * Statement5: traffic matching DSCP AF4, Punt to Non-default vrf
+  * Configure the IBGP peering on ATE2:Port2 to advertise the following
+    * IPv4Prefix1/24, IPv6Prefix1/64 with a local-preference of 200 and VPNH of IPv4Prefix17 and IPv6Prefix17
+  * Start all the Flows from ATE2:Port3 to DUT:Port4, both GUE encaped as well as unencaped.<br>
+  * Expectations:
+    * All GUE encaped traffic received in the Default VRF destined for IPv4Prefix12/28 is decaped and sent back to ATE2:Port3 based on LPM lookup of EBGP learnt routes. Ensure that the DSCP vaule from Outer header is copied to the inner header. Also, ensure that the TTL value of the inner header is accurate and is the same as what it was prior to encap.
+    * Unencaped BE1 traffic is punted to default VRF and then GUE encaped to IPv4Prefix11/28, its TOS Byte updated as per the table above plus, ttl value from the inner header is copied to the outer header.
+    * Unencaped AF1-AF4 traffic received from ATE2:Port3 must be routed to ATE1:Port1 via the DEFAULT VRF in the DUT based on LPM lookup of the IBGP learnt routes from ATE1:Port1.
+    * All traffic flows must be 100% successful.
+    * Run post test Health checks above to ensure no anomalies<br><br><br>
+
+     
+**RT-3.32.3: BE1-AF1 traffic migration to Default VRF, GUE encap and then route to ATE2:Port3**
+  * Please run all the health checks suggested above prior to starting the test as a baseline.
+  * DUT has the following VRF selection policy
+    * Statement1: traffic matching tunnel-destination "IPv4Prefix12/28", Punt to default vrf
+    * Statement2: traffic matching DSCP BE1, Punt to default vrf
+    * Statement3: traffic matching DSCP AF1, Punt to default vrf
+    * Statement4: traffic matching DSCP AF2, Punt to Non-default vrf
+    * Statement5: traffic matching DSCP AF3, Punt to Non-default vrf
+    * Statement5: traffic matching DSCP AF4, Punt to Non-default vrf
+  * Configure the IBGP peering on ATE2:Port2 to advertise the following
+    * IPv4Prefix1/24, IPv6Prefix1/64, IPv4Prefix1/24, IPv6Prefix1/64 with a local-preference of 200 and VPNH of IPv4Prefix17 and IPv6Prefix17
+  * Start all the Flows from ATE2:Port3 to DUT:Port4, both GUE encaped as well as unencaped.<br>
+  * Expectations:
```

Comment:
It would be clearer -- to me -- if these tests were written with an expectation that can be validated at the ATE that perhaps doesn't focus on how the device did the lookup.

Can I suggest something like:

> * For flows A-->B
>    * Follow the IPv[46] route in VRF FOO
>    * Are received at ATE port X with a header of [IP, UDP, XXX] with DSCP 42 and TTL 44.

I think this will make it a bit clearer what we're expecting in each case, and likely makes it much easier for implementation.